build.sbt

```diff
@@ -50,16 +50,16 @@ moveReact := {
 }
 
 lazy val microservice = Project(appName, file("."))
-  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
```

Comment:
How come you removed the SbtSutoBuildPlugin?

Reply:
Because it's now an AutoPlugin, you don't need to explicitly enable it.

Reply:
Fantastic thanks